docs/calibration.md

```diff
@@ -0,0 +1,52 @@
+## Calibrated URDF for manipulation planning
+
+By default we load from [assets/hab_stretch/urdf/planner_calibrated_manipulation_mode.urdf](assets/hab_stretch/urdf/planner_calibrated_manipulation_mode.urdf) for manipulation.
```

Comment:
Jfyi that this link leads to a 404.

Reply:
thanks, it was wrong anyway - moving towards what we are discussing via email, i'm going to use the current URDFs just for unit tests.